content/go/concepts/operators/operators.md

```diff
@@ -0,0 +1,139 @@
+- `>`: Takes two expressions, `(A > B)`, and returns `true` if the left expression is numerically greater than or comes lexically after the right one.
+- `>=`: Takes two expressions, `(A >= B)`, and returns `true` if the left expression is numerically greater than or comes lexically after the right one, or if both expressions are equal.
+
+## Logical Operators
+
+Logical operators return a `true` or `false` value based on the `true` or `false` values of the expressions they operate on. When expressions are numeric, a zero value is treated as `false` and a non-zero value is treated as `true`.
+
+- `&&` AND operator: Takes two expressions, `(A && B)`, and returns `true` if both expressions are `true`.
+- `||` OR operator: Takes two expressions, `(A || B)`, and returns `true` if one of them is `true`.
+- `!` NOT operator: Takes one expression, `!(A)`, and returns `true` if its value is `false`.
+
+## Bitwise Operators
+
+Bitwise operators manipulate the individual bits of their numeric expressions.
+
+- `&` Binary AND Operator: Takes two expressions, `(A & B)`, result will have a bit set if the corresponding bit in each expression is set.
+- `|` Binary OR Operator: Takes two expressions, `(A | B)`, result will have a bit set if either expression has the corresponding bit set.
+- `^` Binary XOR Operator: Takes two arguments, `(A ^ B)`, result wil have a bit set if only one corresponding bit is set between the two expressions.
+- `<<` Binary Left Shift Operator: Takes two arguments, `(A << B)`, result will be the first argument with its bits moved to the left the number of positions specified by the second.
+- `>>` Binary Right Shift Operator: Takes two arguments, `(A >> B)`, result will be the first argument with its bits moved to the right the number of positions specified by the second.
+
+## Assignment Operators
+
+Assignment operators store the value of an expression into a variable. The variable being assigned to is always on the left.
+
+<details>
+  <summary>
+    <code>=</code>: Assigns the value of the expression to the right to the variable on the left.
+  </summary>
+  <code>C = A + B</code> stores the value of <code>A + B</code> into the variable <code>C</code>.
+</details>
+
+<details>
+  <summary>
+    <code>+=</code>: Adds the value on the right to the value in the variable on the left and assigns it to the variable.
+  </summary>
+  <code>C += A</code> is the same as <code>C = C + A</code>.
+</details>
+
+<details>
+  <summary>
+    <code>-=</code>: Subtracts the value on the right from the value in the variable on the left and assigns it to the variable.
+  </summary>
+  <code>C -= A</code> is the same as <code>C = C - A</code>.
+</details>
+
+<details>
+  <summary>
+    <code>*=</code>: Multiplies the value on the right to the value in the variable to the left and assigns it to the variable.
+  </summary>
+  <code>C *= A</code> is the same as <code>C = C * A</code>.
```

Comment:
```suggestion
  <code>C <!-- prettier-ignore -->*= A</code> is the same as <code>C = C * A</code>.
```

Reply:
@SSwiniarski When I ran `lint:md`, I kept getting an error on this line:

```shell
./content/go/concepts/operators/operators.md:89:1 MD037/no-space-in-emphasis Spaces inside emphasis markers [Context: "  <code>C *"]
```

I wonder if we can just place this comment here to fix the testing error.